openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java

```diff
@@ -85,6 +87,9 @@
     private Route route;
     private Service sshService;
 
+    @Inject
+    private Configuration configuration;
```

Comment:
Changed! Thanks for catching this again!

I'm not really happy with my fix though.